lua/lazyvim/plugins/extras/coding/luasnip.lua

```diff
@@ -85,4 +84,12 @@ return {
       },
     },
   },
+
+  -- Luasnip sources for blink
+  {
+    "saghen/blink.cmp",
+    opts = function(_, opts)
+      opts.sources.default = { "lsp", "path", "luasnip", "buffer" }
```

Comment:
this will remove lazydev and possibly AI providers.
Better to just append luasnip and `vim.tbl_filter` snippets out?

Reply:
Tnx didn't think about that. `vim.tbl_filter` does indeed make more sense. Tnx for your suggestion :smile: